src/service.js

```diff
@@ -1,22 +1,30 @@
 let express = require('express');
 let http = require('http');
+let session = require('./session');
+let adb = require('./adb');
 
 let server;
 
-exports.start = function start(port = 8080) {
+exports.start = function start(options = {}) {
   let app = express();
+
+  app.use(session());
+  app.use(adb({ path: options.adbPath }));
```

Comment:
nit: no object spaces (sorry I'm meaning to add linter so this doesn't have to be part of code review)